pkg/util/find_test.go

```diff
@@ -0,0 +1,81 @@
+package util
+
+import (
+	"github.com/magiconair/properties/assert"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"testing"
+)
+
+func TestGetMountedSecretNameInDeployment(t *testing.T) {
+	d := &v1.Deployment{
```

Comment:
This looks like a good candidate for using an actual manifest yaml file instead of defining all the deployment inline :)

Reply:
I dislike the files but I can do it inline for you :)